src/actions.rs

```diff
@@ -57,77 +57,80 @@ impl<'a> Action for Step<'a> {
+                                        .iter()
+                                        .map(|issue| IssueDecorator {
+                                            title: issue.title.clone(),
+                                            number: issue.number,
+                                            html_url: issue.html_url.clone(),
+                                            repo_name: repository
+                                                .full_name
+                                                .split("/")
+                                                .last()
+                                                .expect("Failed to split repository name")
+                                                .to_string(),
+                                            labels: issue
+                                                .labels
+                                                .iter()
+                                                .map(|l| l.name.as_ref())
+                                                .collect::<Vec<_>>()
+                                                .join(", "),
+                                            assignees: issue
+                                                .assignees
+                                                .iter()
+                                                .map(|u| u.login.as_ref())
+                                                .collect::<Vec<_>>()
+                                                .join(", "),
+                                        })
+                                        .collect();
+
+                                    results
+                                        .entry(*name)
+                                        .or_insert(Vec::new())
+                                        .extend(issues_decorator);
+                                }
+                                Err(err) => {
+                                    eprintln!("ERROR: {}", err);
+                                    err.chain()
+                                        .skip(1)
+                                        .for_each(|cause| eprintln!("because: {}", cause));
+                                    std::process::exit(1);
+                                }
                             }
-                            Err(err) => {
-                                eprintln!("ERROR: {}", err);
-                                err.chain()
-                                    .skip(1)
-                                    .for_each(|cause| eprintln!("because: {}", cause));
-                                std::process::exit(1);
+                        }
+
+                        github::QueryKind::Count => {
+                            let count = repository.get_issues_count(&gh, &query).await;
+
+                            match count {
+                                Ok(count) => {
+                                    *context.entry(*name).or_insert(0) += count;
```

Comment:
In particular, note that the idea behind this line that we've shared with you is right but `entry` is not a method of `Context`. We would need to manually implement this behavior. Check https://docs.rs/tera/1.9.0/tera/struct.Context.html for more info.

Reply:
Yeah, I got that error while compiling but did not get the time to dig into it, now that I see `Context` does not have an entry method so esentially I would need to see if *name is present in context if not then enter a zero else increment the count. 
I am still getting used to rust, so I am assuming I write a  different function that takes in the context and then manipulates the context object and increment the count accordingly right : )

Reply:
Right, unsure if a new function will be needed though. I guess you could write that in a couple of lines of code there.

Reply:
You would need to do something like this ...

```rust
                                    let value = if let Some(value) = context.get(*name) {
                                        value.as_u64().unwrap() + count as u64
                                    } else {
                                        count as u64
                                    };

                                    context.insert(*name, &value);
```

Not sure if that will work, please test it out :).